cms/methodology/models.py

```diff
@@ -126,17 +127,24 @@ def related_publications(self) -> "PageQuerySet":
             exclude_non_matches=True,
         )
 
-    def get_formatted_related_publications_list(self, request: HttpRequest | None = None) -> list[dict[str, str]]:
+    def get_formatted_related_publications_list(
+        self, request: HttpRequest | None = None
+    ) -> dict[str, Union[str, "StrPromise", list[dict[str, str]]]]:
         """Returns a formatted list of related internal pages for use with the Design System list component."""
-        items = []
-        for page in self.related_publications:
-            items.append(
-                {
-                    "title": getattr(page, "display_title", page.title),
-                    "url": page.get_url(request=request),
-                }
-            )
-        return items
+        items = [
+            {
+                "title": getattr(page, "display_title", page.title),
+                "url": page.get_url(request=request),
+            }
+            for page in self.related_publications
+        ]
+
+        if items:
+            return {
+                "title": _("Related publications"),
+                "itemsList": items,
+            }
+        return {}
```

Comment:
Non-blocking suggestion, I think this might be more "Pythonic":
```suggestion
        return {"title": _("Related publications"), "itemsList": items} if items else {}
```